packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -107,6 +111,7 @@ class _DropdownMenuItemButton<T> extends StatefulWidget {
   final BoxConstraints constraints;
   final int itemIndex;
   final bool enableFeedback;
+  final BorderRadius? borderRadius;
```

Comment:
Might be a little simpler to make this non-nullable and pass along `widget.borderRadius ?? BorderRadius.zero`